src/graphics/draw/MenuHandler.cpp

```
@@ -16,12 +16,23 @@
 #include "graphics/TFTColorRegions.h"
 #include "graphics/draw/MessageRenderer.h"
 #include "graphics/draw/UIRenderer.h"
+#if (defined(T_DECK_MAX) || defined(_VARIANT_T_DECK_PRO_V1_1)) && \
+    (defined(HAPTIC_FEEDBACK_PIN) || defined(HAS_DRV2605))
+#include "input/HapticFeedback.h"
+#endif
```

Cppcheck (2.21.0)

[error] There is an unknown macro here somewhere. Configuration is required. If PROGMEM is a macro then please configure it.

(unknownMacro)